website/docs/r/lb_app_profile.html.markdown

```diff
@@ -0,0 +1,131 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_lb_app_profile"
+sidebar_current: "docs-vcd-resource-lb-app-profile"
+description: |-
+  Provides a NSX load balancer application profile resource.
+---
+
+# vcd\_lb\_app\_profile
+
+Provides a vCloud Director Edge Gateway Load Balancer Application Profile resource. An application
+profile defines the behavior of the load balancer for a particular type of network traffic. After
+configuring a profile, you associate it with a virtual server. The virtual server then processes
+traffic according to the values specified in the profile.
+
+~> **Note:** This resource does not currently support attaching  Pool and Virtual
+Server certificates. The `enable_pool_side_ssl` only toggles the option, but does not setup
+certificates.
+
+~> **Note:** To make load balancing work one must ensure that load balancing is enabled on edge
+gateway (edge gateway must be advanced).
+This depends on NSX version to work properly. Please refer to [VMware Product Interoperability
+Matrices](https://www.vmware.com/resources/compatibility/sim/interop_matrix.php#interop&29=&93=) 
+to check supported vCloud director and NSX for vSphere configurations.
+
+~> **Note:** The vCloud Director API for NSX supports a subset of the operations and objects defined
+in the NSX vSphere API Guide. The API supports NSX 6.2, 6.3, and 6.4.
+
+
+
+Supported in provider *v2.4+*
+
+## Example Usage 1 (TCP Application Profile)
+
+```hcl
+provider "vcd" {
+  user     = "${var.admin_user}"
+  password = "${var.admin_password}"
+  org      = "System"
+  url      = "https://AcmeVcd/api"
+}
+
+resource "vcd_lb_app_profile" "tcp" {
+  org          = "my-org"
+  vdc          = "my-org-vdc"
+  edge_gateway = "my-edge-gw"
+
+  name = "tcp-app-profile"
+  type = "TCP"
+}
+```
+
+## Example Usage 2 (HTTP Cookie based Application Profile)
```

Comment:
Nice how you have two different-type examples. Are there any other substantially different ways that would benefit the user in even more examples?

Reply:
Tricky to say. Of course I can think of something more - like HTTPS with passthrough. UDP,  but up for votes how many examples are not too many.

Reply:
OK, let's leave two for now. If we start to get question on usage, we can add them later.